### DISCUSSION TITLE
Provide indication that a column is used in a relationship

### DISCUSSION THREAD

**Discussion by mwardclovisusd** [TabularEditor/TabularEditor3#1061] [Ideas]
In visual studio, there is a different icon for columns involved in a relationship. It would be helpful to have the same indication to be able to quickly see if I've missed adding any relationships.

**Comment by otykier** [comment]
Great idea. Would it make sense to have different icons based on whether the column is used on the One or the Many side of a relationship?

**Comment by bernatagulloesbrina** [comment]
you could just borrow the idea from SQL Server and put a small black key if it's PK and in a one to many relationship and a white one if it's on the many side. and then maybe gray if it's many to many 😁

**Comment by corycundy** [comment]
I think it would be great to have the icon updated as this idea suggest as well as discussion #1114 .

I think in addition to this it would be great if the hover over of the column would contain the relationship or relationships that that column is involved in. The only downside I can think of is if a dimension key relates to many facts, the tool tip would be slightly larger. Worst case, this could be on by default and toggled off as a preference setting.

**Reply by mlonsk** [reply]
Hi @corycundy 
Interesting idea, we will take it into consideration.
